packages/mobile/src/screens/profile-screen/ProfileTabs/AlbumsTab.tsx

```diff
@@ -1,42 +1,28 @@
-import { useEffect } from 'react'
-
-import { Status } from '@audius/common/models'
-import { profilePageActions, profilePageSelectors } from '@audius/common/store'
+import { useUserAlbums } from '@audius/common/api'
 import { useIsFocused } from '@react-navigation/native'
-import { useDispatch, useSelector } from 'react-redux'
+import { useSelector } from 'react-redux'
 
 import { CollectionList } from 'app/components/collection-list/CollectionList'
 import { spacing } from 'app/styles/spacing'
 
 import { EmptyProfileTile } from '../EmptyProfileTile'
 import { getIsOwner, useSelectProfile } from '../selectors'
 
-const { getProfileAlbums, getCollectionsStatus } = profilePageSelectors
-const { fetchCollections } = profilePageActions
-
 const emptyAlbums = []
 
 export const AlbumsTab = () => {
-  const { handle, album_count } = useSelectProfile(['handle', 'album_count'])
-  const albums = useSelector((state) => getProfileAlbums(state, handle))
-  const collectionsStatus = useSelector((state) =>
-    getCollectionsStatus(state, handle)
-  )
+  const { handle, album_count, user_id } = useSelectProfile([
+    'handle',
+    'album_count',
+    'user_id'
+  ])
   const isOwner = useSelector((state) => getIsOwner(state, handle ?? ''))
   const isFocused = useIsFocused()
-  const dispatch = useDispatch()
-  const isLoading = collectionsStatus === Status.LOADING
 
-  const shouldFetchAlbums =
-    isFocused &&
-    (album_count > 0 || isOwner) &&
-    collectionsStatus === Status.IDLE
-
-  useEffect(() => {
-    if (shouldFetchAlbums) {
-      dispatch(fetchCollections(handle))
-    }
-  }, [shouldFetchAlbums, dispatch, handle])
+  const { data: albums, isLoading } = useUserAlbums(
```

Comment:
I think `isPending` is recommended?

As far as I understand, `isPending` is true as long as there's no data, whereas `isLoading` is true if there's data or we're re-fetching

Reply:
good to know, thanks